src/test/java/org/opentripplanner/routing/algorithm/filterchain/filters/system/FlexSearchWindowFilterTest.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner.routing.algorithm.filterchain.filters.system;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.opentripplanner.framework.time.TimeUtils.time;
+import static org.opentripplanner.model.plan.TestItineraryBuilder.newItinerary;
+
+import java.time.Instant;
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.opentripplanner.model.plan.PlanTestConstants;
+import org.opentripplanner.model.plan.TestItineraryBuilder;
+
+class FlexSearchWindowFilterTest implements PlanTestConstants {
+
+  private static final Instant LATEST_DEPARTURE_TIME = TestItineraryBuilder
+    .newTime(time("9:20"))
+    .toInstant();
+
+  @ParameterizedTest
+  @ValueSource(strings = { "9:20", "9:21", "13:20" })
+  void keepFlexItinerariesAfterLDT(String startTime) {
+    var edt = "9:20";
+    var subject = new FlexSearchWindowFilter(TestItineraryBuilder.newTime(time(edt)).toInstant());
+
+    var itin = newItinerary(A, time(startTime))
+      .flex(T11_00, T11_30, B)
+      .withIsSearchWindowAware(false)
+      .build();
+
+    assertThat(subject.flagForRemoval(List.of(itin))).isEmpty();
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = { "0:0", "0:01", "9:19" })
```

Comment:
`0:0` is a bit ugly syntax for time.

Reply:
I've made the notation consistent: ac53bc05012ad45f26258f1d376aff977b91f139